src/Compilers/CSharp/Portable/Binder/Semantics/Operators/BinaryOperatorOverloadResolution.cs

```diff
@@ -825,25 +834,95 @@ private static LiftingResult UserDefinedBinaryOperatorCanBeLifted(TypeSymbol lef
                     {
                         continue;
                     }
+
                     var better = BetterOperator(candidates[i].Signature, candidates[j].Signature, left, right, ref useSiteDiagnostics);
-                    if (better == BetterResult.Left)
+                    if (better == BetterResult.Left && candidates[j].Kind != OperatorAnalysisResultKind.Worse)
```

Comment:
> no need to check already worse members.

There is a need to do that, they can still be better then some other candidate that isn't marked Worse yet.